src/cdomains/intDomain.ml

```diff
@@ -101,13 +101,66 @@ module Size = struct (* size in bits as int, range as int64 *)
     try int64_of_big_int y with _ -> raise Not_in_int64
 end
 
-module Interval32 : S with type t = (int64 * int64) option = (* signed 32bit ints *)
+module IntPair =
+struct
+  type t = (int64 * int64) [@@deriving to_yojson]
```

Comment:
Why not simply include the Pair?

Reply:
Already done, I forgot to include this in that commit